planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module.hpp

```diff
@@ -126,6 +129,15 @@ class AvoidanceModule : public SceneModuleInterface
     rtc_interface_right_.clearCooperateStatus();
   }
 
+  void removeCandaiteRTCStatus()
```

Comment:
@tkimura4 Could you fix the typo?
```suggestion
  void removeCandidateRTCStatus()
```